File: Pattern/pattern4/Pattern.java
package Pattern.pattern4;

public class Pattern {
    public static void main(String[] args) {
        pattern(9);
    }

    public static void pattern(Integer n) {
        for (int i = 1; i <= n; i++) {
            for (int j = 1; j <= i; j++) {
                System.out.print(j);
            }
            System.out.println("");
        }
    }
}

File: Pattern/pattern4/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 1; i <= x; i++ {
		for j := 1; j <= i; j++ {
			fmt.Print(j)
		}
		fmt.Println("")
	}
}

func main() {
	pattern(7)
}
